Header header

# дистанция под днищем, [м], [0, inf)
float32 downward